packages/@aws-cdk/aws-chatbot/lib/slack-channel-configuration.ts

```diff
@@ -303,5 +312,17 @@ export class SlackChannelConfiguration extends SlackChannelConfigurationBase {
     this.slackChannelConfigurationArn = configuration.ref;
     this.slackChannelConfigurationName = props.slackChannelConfigurationName;
   }
+
+  /**
+   * Adds a SNS topic that deliver notifications to AWS Chatbot.
+   * @param notificationTopic
+   */
+  public addNotificationTopic(notificationTopic: sns.ITopic): void {
+    this.notificationTopics.push(notificationTopic);
+  }
+
+  private renderTopics(): string[] {
```

Comment:
How about we inline this method? Doesn't seem super helpful (it's only called once).

Reply:
refactored